src/loader/loader.cr

```diff
@@ -12,19 +12,27 @@ module BakedFileSystem
                   reject { |path| File.directory?(path) || !(path =~ /(\/\..+)/).nil? }
 
       files.each do |path|
-        # encoded_path,encoded_mime_type,size,urlsafe_encoded_content
+        # encoded_path,encoded_mime_type,size,compressed_size,urlsafe_encoded_gzipped_content
         entity = [] of String
 
         # File name
-        entity << Base64.strict_encode(path[root_path_length..-1])
```

Comment:
You choose `|` as separator and remove `Base64.strict_encode`. But file name can contain `|` and then it can blow when you load data.